api/specs/webserver/scripts/openapi_projects_crud.py

```diff
@@ -105,10 +105,17 @@ async def list_projects(
     show_hidden: bool = Query(
         default=False, description="includes projects marked as hidden in the listing"
     ),
+    search: str = Query(
+        default=None,
+        description="Multi column full text search",
+        max_length=25,
```

Comment:
Question: why is there a 25 max length here?

Reply:
Just wanted to put some reasonable upper bound, so when user makes some mistake for example when he is copy/pasting something so he does not overload us with a large url.